content/en/agent/basic_agent_usage/windows.md

```diff
@@ -23,26 +23,38 @@ This page outlines the basic features of the Windows Datadog Agent. If you haven
 
 ## Agent installation
 
-Many items can be configured on the command line when installing the Datadog Windows Agent. Each configuration item is added as an install property to the command line. For instance, the following commands install the Agent, configure the Agent configuration file with the `<DATADOG_API_KEY>`, and set the `<HOSTNAME>` and tags.
+**Starting with release `6.11.0`, the Core and APM/Trace components of the Windows Agent run under the `ddagentuser` account, created at install time, instead of running on prior versions under the `LOCAL_SYSTEM` account. If enabled, the Live Process component still runs under `LOCAL_SYSTEM`.**
```

Comment:
I think we need to modify the wording on this a bit...
`... instead of running on prior versions under LOCAL_SYSTEM` account` 

Perhaps
`... instead of running under the `LOCAL_SYSTEM` account, as was the case on prior versions`